R/load_cell_graphs.R

```diff
@@ -232,22 +416,24 @@ LoadCellGraphs.Seurat <- function (
   add_marker_counts = TRUE,
   force = FALSE,
   chunk_length = 10,
+  cl = NULL,
   verbose = TRUE,
   ...
 ) {
 
   # Use default assay if assay = NULL
   if (!is.null(assay)) {
-    stopifnot("'assay' must be a character of length 1" = is.character(assay) & (length(assay) == 1))
+    stopifnot("'assay' must be a character of length 1" = is.character(assay) && (length(assay) == 1))
```

Comment:
What about:
```
# Use default assay if assay = NULL
  assay <- assay %||% DefaultAssay(object)
```

Reply:
Not sure if I follow :-O It's there after the else statement.

Reply:
Ok, nevermind!